Ports/qt6-qt5compat/package.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='qt6-qt5compat'
+version='6.2.3'
+workdir="qt5compat-everywhere-src-${version}"
+useconfigure='true'
+files="https://download.qt.io/official_releases/qt/$(cut -d. -f1,2 <<< ${version})/${version}/submodules/qt5compat-everywhere-src-${version}.tar.xz qt6-qt5compat-${version}.tar.xz 1cf89198cf2cf8a5c15336ccd69fa1f39b779feb64117d6bbf5509c21c123f53"
+auth_type='sha256'
+configopts=(
+    '-GNinja'
+    "-DCMAKE_TOOLCHAIN_FILE=${SERENITY_BUILD_DIR}/CMakeToolchain.txt"
+    '-DCMAKE_CROSSCOMPILING=ON'
+    '-DQT_BUILD_TOOLS_WHEN_CROSSCOMPILING=ON'
+    '-DQT_HOST_PATH=/usr'
+    '-DQT_FEATURE_cxx20=ON'
+)
+depends=(
+    'qt6-qtbase'
+    'libiconv'
+)
+
+QT_HOST_PATH='/usr'
+QT_HOST_CMAKE_PATH="${QT_HOST_PATH}/lib64/cmake"
+QT_HOST_TOOLS='HostInfo CoreTools GuiTools'
+QT_HOST_TOOLS_PATH="${QT_HOST_CMAKE_PATH}/Qt6%s/\n"
+
+configure() {
+    for host_tool in ${QT_HOST_TOOLS}; do
+        if [[ ! -d $(printf $QT_HOST_TOOLS_PATH $host_tool) ]]; then
```

Comment:
What is this even doing? `printf some_string blah blah` just prints `some_string`.

Reply:
Don't know, this comes from qt6-qtbase...

Reply:
cc @MartinBriza

Reply:
it does the same thing as printf in c, see the variable that's ship lied as the first argument - it contains the %s directive so it will basically merge the two strings. 

there was a reason i used printf instead of just simple concatenation but i forgot what it was

Reply:
Ah, damn it, I mixed up `QT_HOST_PATH` and `QT_HOST_TOOLS_PATH`.

That is fine then, sorry for the ping and the confusion. :^)